workitem/link/link_repository.go

```diff
@@ -458,3 +459,30 @@ func (r *GormWorkItemLinkRepository) WorkItemHasChildren(ctx context.Context, pa
 	}
 	return hasChildren, nil
 }
+
+// GetParentID returns parent ID of the given work item if any
+func (r *GormWorkItemLinkRepository) GetParentID(ctx context.Context, ID uuid.UUID) (*uuid.UUID, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "workitemlink", "get", "parent"}, time.Now())
+	query := fmt.Sprintf(`
+			SELECT id FROM %[1]s WHERE id in (
+				SELECT source_id FROM %[2]s
+				WHERE target_id = $1 AND deleted_at IS NULL AND link_type_id IN (
+					SELECT id FROM %[3]s WHERE forward_name = 'parent of'
```

Comment:
@pranavgore09 as we've discussed, this will later be changed to a link type defined by the space template instead of manually having to search for a link type with `parent of` in the forward name.

Reply:
thanks! using in https://github.com/fabric8-services/fabric8-wit/pull/1592/commits/44dd3147d43c80781a4e26055c406bdd1f3fd99a